scripts/remove_after_use/fix_unclaimed_records_for_preprints.py

```diff
@@ -0,0 +1,49 @@
+import sys
+import logging
+
+from framework.auth.core import generate_verification_key
+from website.app import setup_django
+
+setup_django()
+from osf.models import OSFUser
+logger = logging.getLogger(__name__)
+
+
+def add_missing_unclaimed_record(user, node, dry_run):
+    # Get referrer from logs
+    for log in node.logs.filter(action='contributor_added').order_by('date'):
+        if user._id in log.params['contributors']:
+            if log.user_id:
+                referrer = OSFUser.objects.get(id=log.user_id)
+                verification_key = generate_verification_key(verification_type='confirm')
+                record = {
+                    'name': user.given_name,
```

Comment:
I think this should be `user.fullname`. Otherwise, only the first name is shown in the contributor list.

For example, for https://osf.io/4aybw/:

**Before**

![image](https://user-images.githubusercontent.com/2379650/46306884-7a6ab000-c583-11e8-8446-a3ddc207d799.png)

**After**

![image](https://user-images.githubusercontent.com/2379650/46306896-82c2eb00-c583-11e8-9bcd-1390a2157b35.png)